lib/txinput.js

```diff
@@ -30,9 +30,9 @@ class LedgerTXInput {
    * @param {String|Number[]} options.path
    * @param {bcoin.TX|Buffer} options.tx
    * @param {Number} options.index
-   * @param {bcoin.Script|Buffer?} options.redeem - Redeem script for P2SH transactions
+   * @param {bcoin.Script|Buffer?} options.redeem - script for P2SH transactions
    * @param {Buffer?} options.publicKey - raw public key for ring
-   * @param {Boolean} [options.witness=false]
+   * @param {Buffer?} opitons.trustedInput
```

Comment:
Typo: `options`

Reply:
What's a "trusted input" vs. a normal input?

Reply:
trusted input is ledger term for processed previous txs. Users won't probably ever use that.